### PR TITLE
Purchase request api

### PR DESCRIPTION
**Pull request by harishrconcur**
## Pull Request Checklist

**Complete and check all items that are applicable.**

- [ ] Proofread documentation changes for spelling and grammatical errors
- [ ] Used Markdown code blocks for all applicable examples using code
- [ ] Used relative links when linking to other documentation on Developer Center
- [ ] If creating new API reference pages, followed the guidelines stated [in the wiki](https://github.com/concur/developer.concur.com/wiki/Creating-API-Reference-pages)
- [ ] If moving pages, added [page redirects](https://github.com/concur/developer.concur.com/wiki/Adding-Redirects) so users are redirected to the new page

### CODE REVIEW

**Comment by charlieconcur** [comment]
@harishrconcur Please remove all references to `REST` or `Representational State Transfer` from the docs as the API isn't RESTful. Afterwards, please have a team member peer review the documentation and give a thumbs up -- then I will merge.

**Comment by harishrconcur** [comment]
@charlieconcur These are V4 REST API's for Purchase Request that we are working on.

**Comment by charlieconcur** [comment]
Yes, I understand this is part of the v4 'wave' of APIs but it does not fit the definition of being RESTful -- so we shouldn't call it REST.

File: src/api-reference/invoice/purchase-request.markdown [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+
+|Name | Type | Format | Description
+|-----|------|--------|------------
+|`errorCode`|`string`|`-`|A code that indicates a specific error code indicating why a particular field had failed validation
+|`errorMessage`|`string`|`-`|A description of the error.  
+
+
+## <a name="error-codes"></a>Error Codes  
+
+The REST API's error responses
```
Comment (charlieconcur) [inline src/api-reference/invoice/purchase-request.markdown]: The term 'REST' should be removed here.

**Comment by charlieconcur** [comment]
@harishrconcur Please make the edits as requested. Thanks!

**Comment by charlieconcur** [comment]
@harishrconcur Ping.

**Comment by harishrconcur** [comment]
@charlieconcur I will be making these changes in next few days and get back. Thanks for follow up

**Comment by harishrconcur** [comment]
@charlieconcur we are done with my pending changes implemented changes suggested by you. Please take a look at it and merge. Thanks.

**Pull request merged**